Makefile

```diff
@@ -1,9 +1,10 @@
 default: clean build
 
 EXTENSIONS = \
- 	CustomScript \
- 	DSC \
- 	OSPatching \
```

Comment:
EMACS flagged my spacing in the Makefile, and I **always** listen to EMACS. :grinning: